5-VM-Windows-1-NodeTypes-Secure-NSG/README.md

```diff
@@ -7,7 +7,7 @@
     <img src="http://armviz.io/visualizebutton.png"/>
 </a>
 
-This template allows you to deploy a secure 5 node, Single Node Type Service Fabric Cluster running Windows Server 2016 Datacenter with containers on a Standard_D2_v2 Size Virtual Machine Scale set with Azure Diagnostics turned on and network security groups enabled. This was deployed by product group successfully in Jan 2019.
+This template allows you to deploy a secure 5 node, Single Node Type Service Fabric Cluster running Windows Server 2022 Datacenter on a Standard_D2_v2 Size Virtual Machine Scale set with Azure Diagnostics turned on and network security groups enabled. This was deployed by product group successfully in Jan 2019.
```

Comment:
2019? The statement can be removed.
SKU size on Azure Virtual Machine Scale Set

Reply:
fixed all occurrences